tests/test_observable.py

```diff
@@ -18,49 +18,24 @@
 from pennylane import numpy as np
```

Comment:
Given the scope of this refactor, I'm surprised there are minimal updates to this test file :laughing: